.travis.yml

```diff
@@ -24,15 +33,29 @@ cache:
   directories:
   - node_modules
 before_install:
+- if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then openssl aes-256-cbc -K $encrypted_ddb18e52ae01_key -iv $encrypted_ddb18e52ae01_iv -in id_rsa_downcloud_cozy-settings.enc -out /tmp/id_rsa_downcloud_cozy-settings -d; fi
+- if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then eval "$(ssh-agent -s)"; fi
+- if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then chmod 600 /tmp/id_rsa_downcloud_cozy-settings;
+  fi
+- if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then ssh-add /tmp/id_rsa_downcloud_cozy-settings;
+  fi
```

Comment:
We should handle this part directly in cozy-app-publish, but not blocking for now